R/showAreaAndBarPlotsPlus.R

```diff
@@ -0,0 +1,42 @@
+#' Show Area and Bar Plots Utilizing '+'-Notation
+#'
+#' Automatically infers disaggregation of a variable using the '+'-notaion and
+#' calls \code{\link{showAreaAndBarPlots}}.
+#'
+#' The function requires \code{data} to have a character column named
+#' \code{varplus} containing variable names that use the '+'-notaion. The
+#' function searches for values in \code{varplus} that start with \code{tot}
+#' followed by \code{"|"}, \code{plusNum}-times \code{"+"}, and \code{"|"}.
+#' These variables are then used in a call to \code{\link{showAreaAndBarPlots}}.
+#'
+#' @param tot A single string. A total value to be shown in the area plots.
+#' @param plusNum A single number. Number of "+"symbols for disaggregation.
+#' @inheritParams showAreaAndBarPlots
+#' @return \code{NULL} is returned invisible.
```

Comment:
`return(invisible(NULL))` is the R way of returning nothing, so I suggest to not document it.
```suggestion
```

Reply:
Ok, I can remove that in all `show...`-functions. But then I would want to add something like "The function does not return the plot" in the documentation, because without stating the return value explicitly, there is a high chance that it will be misunderstood. And considering that, I wonder whether it wouldn't be clearer to keep the `@return`  statement. What do you think, @pfuehrlich-pik ?

Reply:
Good point, I can see that. By the way, what speaks against returning the plot? :slightly_smiling_face:

Reply:
The function produces multiple plots, arranges them, an shows them. Returning a list of plots would require the user to do the arranging, which would mean many additional copy-and-paste-lines in the compareScenarios script. One could probably and with some effort return an object that contains multiple pages of plots in an arranged way so that it is correctly shown via `print()` and does not require additional lines of code. But this seems unnecessarily complicated to be able to obtain an object that is very difficult to change (as it will not be a simple ggplot-object).

Reply:
I was just thinking about returning (invisibly) the exact object that you `print`, without changing anything else. It's at least something to work with if users want to try to modify it, but maybe it's not useful, I trust your decision here :slightly_smiling_face: